tests/integration/cairo_path/contracts/external_lib/foo.cairo

```diff
@@ -0,0 +1 @@
+fn foo() {}
```

Comment:
Can we test a scenario when this lib also has another dependency

Reply:
[add a case for nested deps](https://github.com/software-mansion/protostar/pull/1433/commits/c2149cb2949adf6d37a53400d6d2df561dc7b945)